README.md

```diff
@@ -118,10 +118,15 @@ Typst's CLI is available from different sources:
   - macOS: `brew install typst`
   - Windows: `winget install --id Typst.Typst`
 
-- If you have a [Rust][rust] toolchain installed, you can also install the
-  latest development version with
-  `cargo install --git https://github.com/typst/typst --locked typst-cli`. Note that this
-  will be a "nightly" version that may be broken or not yet properly documented.
+- If you have a [Rust][rust] toolchain installed, you can also install a specific version with:
```

Comment:
Here you say "a specific version" and give "v0.11.0" as an example. I think it could be better like this:

```suggestion
- If you have a [Rust][rust] toolchain installed, you can also install a specific version with `--tag`. For example, you can install v0.11.0 with the command below.
```

Reply:
Ideally, we could phrase this in a way that doesn't require me updating the README when making a release.